### DISCUSSION TITLE
Sentencizer Exception: Error while initializing BPE: Token `Ċ` out of vocabulary

### DISCUSSION THREAD

**Discussion by roelvanderburg** [explosion/spaCy#6529] [Help: Coding & Implementations] [answered]
Thanks for the quick reaction. Running above code to create a "train.spacy" file and running command:

``
spacy train config.cfg --paths.train train.spacy --paths.dev dev.spacy -o output_dir
``

Unfortunately gives an

``
Exception: Error while initializing BPE: Token `Ċ` out of vocabulary
``

_I found very little information about this error. It might be some encoding issue ?_

_Originally posted by @adrianeboyd in https://github.com/explosion/spaCy/issues/6382#issuecomment-726064716_

**Comment by adrianeboyd** [comment, marked as answer]
See the original issue #6382 for more context.

I suspect you ended up with a transformers pipeline because of the current `init config` defaults and not because you picked it on purpose? A transformers pipeline is probably complete and total overkill for this particular use case, and this error may just be something like a library version mismatch related to some transformers dependencies? I'm not sure without more info.

If your goal is just to train a `senter` model, try this for the config instead, which uses spacy's default tok2vec instead:

```bash
spacy init config config.cfg -p senter -l nl -C
```

**Comment by CedricMingneau** [comment]
I know I'm a bit late to answer, but since I ran into the same error with my (ner) pipeline, I figured I'd add my two cents for later reference.

I created my config file using the Quickstart webtool that can be found in the spaCy usage guide (https://spacy.io/usage/training#quickstart). If you select Dutch as the language and decide to use a Transformer (GPU), it'll load the "pdelobelle/robbert-v2-dutch-base" model from huggingface. According to the following issue on the huggingface github issuetracker (https://github.com/huggingface/transformers/issues/9290) there seems to be some kind of bug in this model. Last time I checked, this wasn't fixed.

I ended up using another dutch model called BERTje ("GroNLP/bert-base-dutch-cased"). Simply swapping the models fixed the error.

**Reply by ines** [reply]
Thanks for the update, this is really helpful! (For the recommendations, we typically went with the most-downloaded models and ran some basic tests to make sure they work and load. So we probably didn't hit that bug then.) I'll update the quickstart recommendations to use `GroNLP/bert-base-dutch-cased` for now 👍
